arch/cpu/gecko/dev/efr32-radio-buffer.c

```diff
@@ -44,7 +44,6 @@ has_packet(void)
 {
   return rx_buf[next_read].len > 0;
 }
```

Comment:
These files are added in this PR, can you squash this commit with the commit that added the files so the files never have the wrong code-style?

Reply:
Done